### PR TITLE
Clear prettier error with `prettier-ignore` comment

### PR DESCRIPTION
**Pull request by julieg18**
* stops prettier from incorrectly formatting line in docs/command-ref/repro (#3313)

### CODE REVIEW

File: content/docs/command-reference/repro.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
 command [options](#options), such as `--single-item` or `--all-pipelines`.
 
+<!-- prettier-ignore -->
 > Note that stages without dependencies nor outputs are considered [always
```
Comment (julieg18) [inline content/docs/command-reference/repro.md:55]: If we don't add a comment, prettier adds a `>` at the end of line 54.

**Review by yathomasi — APPROVED** [review]
Looks good to me.

This unique issue seems to happen when the bracketed link falls on line wrap. So, I don't see a way to fix this on a prettier config. 
Changing [proseWrap](https://prettier.io/docs/en/options.html#prose-wrap) would affect other.

**Comment by shcheklein** [comment]
Hmm. Restyle is using prettier also. pre-commit and Restyle must be consistent. If they are not- that'a a bug to my mind. Or am I missing something? 

We've never so far had to add prettier-ignore as far as I remember.

**Comment by jorgeorpinel** [comment]
> pre-commit and Restyle must be consistent

They are. The pre-commit hook tries to do the same. I had to `git commit -n` in https://github.com/iterative/dvc.org/pull/3327#issuecomment-1057357682.

**Comment by jorgeorpinel** [comment]
> This unique issue seems to happen when the bracketed link falls on line wrap

Is that somehow valid or should we maybe report the issue to https://github.com/prettier/prettier?

**Comment by jorgeorpinel** [comment]
> We've never so far had to add prettier-ignore as far as I remember.

p.s. we do have one other prettier-ignore block in https://raw.githubusercontent.com/iterative/dvc.org/master/content/docs/user-guide/project-structure/pipelines-files.md

**Comment by jorgeorpinel** [comment]
Correct. Full context:

1. Prettier wands to add a `>` symbol at the end of the line [in an edge case situation](https://github.com/iterative/dvc.org/pull/3313#pullrequestreview-897852863), which seems like a bug.
2. To avoid that I forcibly skipped the hook locally (in https://github.com/iterative/dvc.org/pull/3287/commits/4f8f889948ebdbdcbaea502755895a36cff7c9c4), but Restyled created its PR since Prettier failed upstream.
3. I Ignored the Restyled PR and merged the original (which closed the Restyled one), thinking we wouldn't have to worry about it after that PR.
4. Since the GH checks actually check formatting for the whole code base now (since https://github.com/iterative/dvc.org/pull/2624), merging that line without Prettier's formatting actually caused every PR after to create Restyled ones.
5. This PR tells Prettier to ignore the line to avoid 4.

**Comment by shcheklein** [comment]
Thanks, Jorge. Okay, I see they are aligned and both of them have the same bug. Have we raised the bug in the prettier repo to track? Also, can we put a comment with a link to this discussion somewhere nearby the ignore pragma?

**Comment by julieg18** [comment]
> Thanks, Jorge. Okay, I see they are aligned and both of them have the same bug. Have we raised the bug in the prettier repo to track? Also, can we put a comment with a link to this discussion somewhere nearby the ignore pragma?

Looks like there is already [an issue opened!](https://github.com/prettier/prettier/issues/9099)!

**Comment by jorgeorpinel** [comment]
Great. I subscribed to the issue for now.

**Comment by casperdcl** [comment]
The PR's been open since Sept 2020 (https://github.com/prettier/prettier/pull/9125) so prob not being merged soon.

**Pull request merged**